### ISSUE TITLE
Where is the pretrained model on KITTI ?

### ISSUE THREAD

**Issue by cgbcgb** [overlappredator/OverlapPredator#19]
Thanks for your great work, I have two questions:

1. I Notice there is no pretrained model on KITTI in https://share.phys.ethz.ch/~gsg/Predator/weights.zip, don't you want to make it  public for forgot it ?

2.and there is no file : configs/train/kitti.yaml, and you didn't mention that training network in kitti dataset, only mentioned evaluating Predator on KITTI test set in README.md, So you have not make  the training code/config files on kitti dataset public ?

**Comment by ShengyuH** [comment]
hi, 

Thanks for your interest. To answer your question:
1. I will make it public
2. I will provide the config file 

Please see: https://github.com/overlappredator/OverlapPredator/issues/15#issuecomment-862404089, and https://github.com/overlappredator/OverlapPredator.Mink/issues/3. I am busy with other projects BUT will update them ASAP. 

best,
Shengyu

**Comment by cgbcgb** [comment]
OK, thanks a lot.

**Comment by ShengyuH** [comment]
hi,

1. I have updated KITTI pretrained models
2. For training, I use the curriculum training strategy. Specifically, I start with max_points in circle loss being 512, then increase it to 1024 and 1536, this helps learn sharp features in the end. 

Best,
Shengyu

**Comment by cgbcgb** [comment]
> hi,
> 
> 1. I have updated KITTI pretrained models
> 2. For training, I use the curriculum training strategy. Specifically, I start with max_points in circle loss being 512, then increase it to 1024 and 1536, this helps learn sharp features in the end.
> 
> Best,
> Shengyu

Thank you : )
